### PR TITLE
Introduce stylelint and autoprefixer

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# https://github.com/browserslist/browserslist#readme
```
Comment (XhmikosR) [inline .browserslistrc]: I used what we use in Bootstrap v4, but we can adapt it later

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   ],
   "dependencies": {
     "args": "^5.0.1",
+    "autoprefixer": "^9.7.0",
```
Comment (emedvedev) [inline package.json]: Shouldn't we rather move this (and other modules that are only needed during the build) to devDependencies as well?

Comment (XhmikosR) [reply package.json]: Yeah, but this is simpler for deployments.

Comment (XhmikosR) [reply package.json]: We can revert to the previous behavior and move the devDeps. But if one has set production on Heroku, we need that extra step to explicitly install the devDeps.

**Comment by emedvedev** [comment]
When you have the time, please resubmit the PR to trigger the Now deployment :)

**Comment by XhmikosR** [comment]
I'd say better protect the branches with CI too and see what's needed for Now to deploy for outsiders' PRs :)

**Comment by emedvedev** [comment]
@XhmikosR Now won't deploy outsiders' PRs without authorization, because secrets are used.

**Comment by emedvedev** [comment]
No option to force it as far as I can tell :(

**Comment by XhmikosR** [comment]
@emedvedev there's definitely an option :P In the same place you set Now and lgtm to be required.

And how is PR deployments to work for outsiders? Manually?

**Comment by XhmikosR** [comment]
BTW the deployment errors out with 502 in this PR.

**Comment by emedvedev** [comment]
Manually, yes :( https://zeit.co/docs/v2/advanced/now-for-github#deployment-authorizations-for-forks

**Comment by emedvedev** [comment]
Seems to be working well ❤️

**Comment by XhmikosR** [comment]
Just make sure you enable Node 10 and Node 12 to pass :)

**Review by emedvedev — APPROVED** [review]

**Comment by XhmikosR** [comment]
You shouldn't have squashed this :/

**Comment by emedvedev** [comment]
Oops, sorry! I'm a bit overwhelmed with the amount of changes, so some things slip my mind :) Gotta be more careful.

**Review by johnaAr555 — CHANGES_REQUESTED** [review]
.browserslistrc

**Review by johnaAr555 — APPROVED** [review]

**Pull request merged**